rest/model/admin.go

```diff
@@ -1103,6 +1103,7 @@ type APIServiceFlags struct {
 	TaskLoggingDisabled        bool `json:"task_logging_disabled"`
 	CacheStatsJobDisabled      bool `json:"cache_stats_job_disabled"`
 	CacheStatsEndpointDisabled bool `json:"cache_stats_endpoint_disabled"`
+	TaskReliabilityDisabled    bool `json:"task_reliability_disabled"`
```

Comment:
I toyed with the idea of making this a `TaskReliabilityEnabled` flag. This would mean that it is disabled by default but it would be the opposite sense of the other service flags.

Reply:
If you're intending this to be a switch to turn on/off a feature when it's ready, then I think you should make it that (and not put it as part of the service flags). These are to turn off parts of the system due to external outages